IceCreamShop.playground/Contents.swift

```diff
@@ -1 +1,103 @@
+struct Flavor: Equatable {
```

Comment:
Nice job figuring out how to adopt the Equatable protocol, when we haven't even covered protocols yet. I take it you did this for using .contains() in your orderCone function.